distribution/docs/src/main/resources/jbake.properties

```diff
@@ -1,4 +1,4 @@
-site.host=http://codice.org/ddf
+site.host=http://localhost:8993
```

Comment:
❓ What was happening with the previous site host value?

Reply:
Nothing, because this isn't used to generate a static site, but there must be a value present. We are using jdoc to create an intermediate asciidoctor file, so this is never hosted.